src/shared/components/query/QueryStore.ts

```diff
@@ -1190,7 +1190,11 @@ export class QueryStore
 	                return "Please enter one or more gene symbols.";
 	            }
             }
-        }
+        } else {

```

Comment:
i wish we could understand what changed to cause this logic to fail.

Reply:
yes, its that the check for empty oql is gated by the check on line 1149 for nonempty list of selected profile ids, and I guess for multiple study queries, you dont select profile ids explicitly so that list is empty